services/frontend-v3/src/components/descriptionCard/DescriptionCardView.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import DescriptionText from "../descriptionText/DescriptionText";
+
+const DescriptionView = ({ data }) => {
+  /**
+   * Generate Competencies Tag List
+   *
+   * Generate a list of Competency Tags
+   * If no competencies are found for the profile then display friendly message
+   */
```

Comment:
Could you put the comment outside the function? Or else it won't show when you hover on that DescriptionView variable

Reply:
removed accidental comment